dao/src/main/resources/sql/schema-entities.sql

```diff
@@ -253,6 +253,15 @@ CREATE TABLE IF NOT EXISTS entity_view (
     additional_info varchar
 );
 
+CREATE TABLE IF NOT EXISTS entity_profile(
+    id              varchar(31) NOT NULL CONSTRAINT entity_profile_pkey PRIMARY KEY,
+    name            varchar(255),
+    tenant_id       varchar(31),
+    entity_type     varchar(255),
+    profile         varchar,
+    additional_info varchar
```

Comment:
Please add Unique Constraint as discussed.

Reply:
(and corresponding tests)

Reply:
(Name, TenantId, EntityType)

Reply:
added